lib/vtls/schannel_verify.c

```diff
@@ -467,6 +427,83 @@ static DWORD cert_get_name_string(struct Curl_easy *data,
   return actual_length;
 }
 
+BOOL hostname_is_ip(PCRYPT_DATA_BLOB ip_blob,
```

Comment:
No need to create this function, `Curl_host_is_ipnum` already exists.

Reply:
> No need to create this function, Curl_host_is_ipnum already exists.

Nope. `hostname_is_ip` is a dual purpose functional that returns TRUE if the host name is an ip address, and more importantly it returns the **converted binary form** in `ip_blob` buffer that is used in later steps. I'm not going to change that.

Reply:
Then I think maybe the function name can be improved a little to better suggest that, and it needs to be made static.

I would also prefer using `Curl_inet_pton()` instead of the windows specific `InetPtonA()` call and using `struct in_addr` and `struct in6_addr` instead of `IN_ADDR` and `IN6_ADDR`. In other words we should at least try to make the code look like generic internet code instead of using magic windows ways.